org.eu.encom.matrique.yaml

```diff
@@ -0,0 +1,24 @@
+id: org.eu.encom.matrique
+runtime: org.kde.Platform
+runtime-version: '5.11'
+sdk: org.kde.Sdk
+command: matrique
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+  - --socket=wayland
+  - --device=dri
+  - --filesystem=home
+  - --filesystem=/tmp
```

Comment:
Does it work, if you replace this with `--env=TMPDIR=$XDG_CACHE_HOME`?

Reply:
Sorry but that doesn't work. The file can't even write to $XDG_CACHE_HOME dir.

Reply:
Will it work if I change upstream code and let file download in "~/.cache/<APPNAME>" instead?

Reply:
It worked. I don't believe it.